src/main/java/org/intellij/sdk/codesync/state/PluginState.java

```diff
@@ -1,5 +1,7 @@
 package org.intellij.sdk.codesync.state;
 
 public class PluginState {
-    public String value;
+    public boolean isAuthenticated;
+    public boolean isRepoInSync;
```

Comment:
When isRepoInSync is false, we need to show "Connect you repo with CodeSync" along with Connect button that will trigger the init process (maybe we are doing that already)

Reply:
Yes, that is already happening.

Reply:
So it becomes something like
```
if (not logged-in) {
	// Show Welcome to CodeSync with Join button
	// Join will trigger Auth process
	return
} else if (repo is not in config.yml) {
	// show "Connect you repo with CodeSync" along with Connect button that will trigger the init process
	return
}

// Show Repo X is in sync with CodeSync, with View on web and Unsync options
```